metricflow/test/model/dbt_cloud_parsing/test_dbt_metric_to_dimensions_rules.py

```diff
@@ -66,7 +85,7 @@ def test_dbt_filters_to_dimensions(dbt_metrics: Tuple[MetricNode, ...]) -> None:
         not issues.has_blocking_issues
     ), f"DbtFiltersToDimension raised issues went it shouldn't have: {issues.to_pretty_json()}"
     for dbt_metric in dbt_metrics:
-        if dbt_metric.filters:
+        if CALC_METHOD_TO_METRIC_TYPE[dbt_metric.calculation_method] != MetricType.DERIVED and dbt_metric.filters:
```

Comment:
This filter feels so clumsy but I have nothing better to offer. Hm.

Reply:
Yeaaaaaaa 🙃 A dbt MetricNode doesn't come with a guaranteed shape. It'd be nice if given a type we could have assurances of other attributes being set or not, but we dont. Technically derived dbt metrics can have filters specified, but I believe they don't currently actually do anything.